### PR TITLE
add `Logiee`

### PR DESCRIPTION
**Pull request by h-ohta**
These are S1 products.

### CODE REVIEW

File: cspell/.cspell.json [hunk 1]
```diff
@@ -560,6 +560,8 @@
         "linuxcan",
         "LIVOX",
         "LLVM",
+        "Logiee_S-TC",
+        "Logiee_SS",
```
Comment (kmiya) [inline cspell/.cspell.json:564]: Thanks! But the words are split by `_` and `-` when checking, and the words less than four characters are ignored. So `Logiee` would be the only one we need.
```suggestion
        "Logiee",
```

Comment (h-ohta) [reply cspell/.cspell.json:564]: Thanks for your review. I'll fix it

Comment (kmiya) [reply cspell/.cspell.json:564]: FYI: Just clicking the commit suggestion button would save your time.
image

**Comment by kmiya** [comment]
Thank you!

**Pull request merged**